Dockerfile.tmpl

```diff
@@ -177,12 +181,13 @@ RUN JAXVER=$(pip freeze | grep -e "^jax==") && \
         pandas \
         polars \
         flax \
-        "${JAXVER}" && \
+        "${JAXVER}"
+
+RUN apt-get install -y default-jre
```

Comment:
We call `/tmp/clean-layer.sh` to ensure the apt-cache is cleared at the end of the docker layer, otherwise, this makes this docker image layer larger and increase the total image size.